docs/getting-started/grouped-nav-ui.asciidoc

```diff
@@ -0,0 +1,19 @@
+[[grouped-nav-ui]]
+= Grouped navigation
+
+Grouped navigation is a new navigation UI for {elastic-sec} that groups related pages and highlights commonly visited areas for a streamlined experience. 
+
+NOTE: By default, the {security-app} uses the legacy navigation UI. To enable grouped navigation, go to *{kib}* -> *{stack-manage-app}* -> *Advanced Settings*, then turn on `securitySolution:enableGroupedNav`.
+
+[role="screenshot"]
+image::images/grouped-nav-ui.png[width=75%][height=75%][Overview of grouped navigation UI]
+
+The side navigation menu contains both direct links and category-based groups, identified by the group icon (image:images/group-icon.png[Group icon,16,16]).
+
+* Select top-level links to go directly to pages that are essential for detection and investigation workflows (*Alerts*, *Timelines*, and *Cases*).
```

Comment:
```suggestion
* Click top-level links to go directly to pages that are essential for detection and investigation workflows (*Alerts*, *Timelines*, and *Cases*).
```
I think "select" introduces a moment of ambiguity where the reader is wondering if you mean select as in "specified" or "some" vs the intended meaning of "click on"